lib/resources/IssuerFraudRecords.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+var StripeResource = require('../StripeResource');
+var stripeMethod = StripeResource.method;
+
+module.exports = StripeResource.extend({
+  path: 'issuer_fraud_records',
+  includeBasic: ['list', 'retrieve'],
+
+  retrieveFromCharge: stripeMethod({
+    method: 'GET',
+    path: function(urlData) {
+      return '?charge=' + urlData.chargeId;
```

Comment:
@brandur @matt-stripe any advice here? This still isn't right, because then the url is /v1/issuer_fraud_records/?charge=.... whereas I want it to be /v1/issuer_fraud_records?charge=....

I haven't found another example of this ^ but please do direct me if there is one

Reply:
I also would actually like the test to be `data: {charge: 'ch_123456789'}` versus doing the urlparams. At least, I think that's what we'd prefer to test here